manifests/server/rack.pp

```diff
@@ -16,9 +16,13 @@
   $rack_arguments = $::puppet::server_rack_arguments,
   $user           = $::puppet::server_user,
   $vardir         = $::puppet::vardir,
+  $touch_cmd      = $::osfamily ? {
+    /^(FreeBSD|DragonFly)$/ => '/usr/bin/touch',
+    default                 => '/bin/touch'
```

Comment:
I'm not a fan of this complex logic in class parameters. Why not use ```touch``` as command and set a ```path``` which includes those directories?